packages/node_modules/@webex/widget-speed-dial/src/SpeedDialForm.tsx

```diff
@@ -294,7 +298,10 @@ export const SpeedDialForm = ({
                   id={name}
                   name={name}
                   defaultValue={value}
+                  autoComplete="off"
                   label={t('form.lastName.label')}
+                  aria-label={t('form.lastName.placeholder')}
```

Comment:
Could you please assign aria-label as value instead of aria-label={t('form.lastName.placeholder')}?

Reply:
Okay